scripts/industrial-edge-image-copy.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+
+LOGFILE=/tmp/skopeo-copy.log
+
+# Function log
+# Arguments:
+#   $1 are for the options for echo
+#   $2 is for the message
+#   \033[0K\r - Trailing escape sequence to leave output on the same line
+function log {
+    if [ -z "$2" ]; then
+        echo -e "\033[0K\r\033[1;36m$1\033[0m"
+    else
+        echo -e $1 "\033[0K\r\033[1;36m$2\033[0m"
+    fi
+}
+
+function parse_yaml {
```

Comment:
Maybe asking for yq to be installed and fetch the few keys we need is simpler?

Reply:
Agreed ... let's refactor afterwards